src/App.js

```diff
@@ -0,0 +1,104 @@
+import CircularProgress from '@material-ui/core/CircularProgress'
+import Container from '@material-ui/core/Container'
+import makeStyles from '@material-ui/core/styles/makeStyles'
+import React, { useState, useEffect } from 'react'
+import NavBar from './components/NavBar'
+import ErrorSnackbar from './components/ErrorSnackbar'
+import { API_ROOT, get, params } from './api'
+import { styles } from './theme/styles'
+
+// Lazy load articles for faster page render time
+const ArticleGridListLazy = React.lazy(() => import('./components/ArticleGridList'))
+
+const useStyles = makeStyles(styles)
+
+const filterArticles = (articles, keyword) => {
+  // consider articles.results array:
+  // filter along title, byline, section
+  return articles.filter((article) => {
+    const lck = keyword.toLowerCase()
```

Comment:
What's `lck` stand for? Or rather than having to parse out the meaning, consider using a more descriptive variable and letting the minifier worry about making it small.

Reply:
It stands for `lowerCaseKeyword` – and I agree, it ought to be more descriptive for legibility sake, especially since it will be minified once built anyhow.